<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMenuCalib</class>
 <widget class="QWidget" name="FormMenuCalib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormMenuCalib</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox::indicator, QRadioButton::indicator {
width: 24px;
height: 24px;
}</string>
  </property>
  <widget class="QWidget" name="dou_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1004</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget {
background: white;
border: 1px solid blue;
border-radius: 20px;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="dou_bg">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>340</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>720</width>
        <height>340</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: gray;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QImageLineEdit" name="val_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="whatsThis">
    <string>砝码重量：?</string>
   </property>
   <property name="text">
    <string>砝码重量：</string>
   </property>
  </widget>
  <widget class="QImageLineEdit" name="val_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>580</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="whatsThis">
    <string>满量程：?</string>
   </property>
   <property name="text">
    <string>满量程：</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>350</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget {
background: gray;
border: 1px solid white;
border-radius: 15px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="weight">
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 60px;
color: blue;
}
QLabel:!enabled {
color: red;
}</string>
      </property>
      <property name="text">
       <string>1234.5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>470</y>
     <width>384</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget {
background: gray;
border: 1px solid white;
border-radius: 15px;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="c_init">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>初始标定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="c_anew">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>重新标定</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CheckBox" name="c_all">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>标定所有</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="zero">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>置零</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="full">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>满度</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QImageLineEdit" name="val_x_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="whatsThis">
    <string>砝码重量：?</string>
   </property>
   <property name="text">
    <string>砝码重量：</string>
   </property>
  </widget>
  <widget class="QImageLineEdit" name="val_x_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>580</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="whatsThis">
    <string>满量程：?</string>
   </property>
   <property name="text">
    <string>满量程：</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>250</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QWidget {
background: gray;
border: 1px solid white;
border-radius: 15px;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QRadioButton" name="zdsen_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>振动1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="zdsen_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>振动3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="zdsen_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>振动2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="zdsen_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>振动4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QImageLineEdit" name="val_nosel_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="whatsThis">
    <string>砝码重量：?</string>
   </property>
   <property name="text">
    <string>砝码重量：</string>
   </property>
  </widget>
  <widget class="QImageLineEdit" name="val_nosel_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>580</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="whatsThis">
    <string>满量程：?</string>
   </property>
   <property name="text">
    <string>满量程：</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_zd" native="true">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>250</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QImageLineEdit" name="val_zd_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="whatsThis">
    <string>砝码重量：?</string>
   </property>
   <property name="text">
    <string>砝码重量：</string>
   </property>
  </widget>
  <widget class="QImageLineEdit" name="val_zd_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>580</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="whatsThis">
    <string>满量程：?</string>
   </property>
   <property name="text">
    <string>满量程：</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QImageLineEdit</class>
   <extends>QLineEdit</extends>
   <header>MWidget.h</header>
  </customwidget>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>CheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
